ioctl/cmd/action/action.go

```diff
@@ -91,8 +91,6 @@ func (m *sendMessage) String() string {
 func init() {
 	ActionCmd.AddCommand(actionHashCmd)
 	ActionCmd.AddCommand(actionTransferCmd)
-	ActionCmd.AddCommand(actionDeployCmd)
-	ActionCmd.AddCommand(actionInvokeCmd)
```

Comment:
have we tested contractDeploy/Invoke command to deploy contract and invoke calls? and new contract command is equivalent with these 2?

Reply:
> have we tested contractDeploy/Invoke command to deploy contract and invoke calls? and new contract command is equivalent with these 2?

we have tests for contract Deploy/Invoke command and examples can be found in [https://github.com/iotexproject/iotex-docs/blob/master/docs/ioctl.md](url). And issue #2188 claims that these are equivalent with action commands.